src/screens/Profile/Login/ConfirmCode.tsx

```diff
@@ -64,6 +69,20 @@ export default function ConfirmCode({navigation, route}: ConfirmCodeProps) {
     }
   };
 
+  // User might be automatically logged in with Firebase auth, but only on Android
+  // Check user from state and see if it is updated while we wait
+  useEffect(() => {
+    if (user?.phoneNumber) {
+      // OK To use toast here, as this is Android only for now
+      ToastAndroid.showWithGravity(
+        t(LoginTexts.confirmCode.autoVerifyToast),
+        ToastAndroid.SHORT,
+        ToastAndroid.CENTER,
+      );
+      navigation.navigate(afterLogin.routeName as any, afterLogin.routeParams);
+    }
+  });
```

Comment:
You should provided a dependency list to make this useEffect not run on every render. With the current code the dependency list should contain `user?.phoneNumber`, or `authenticationType` if the code is changed by the above proposal.

Reply:
Fixed, thanks for the tip!